app/controllers/users_controller.rb

```diff
@@ -23,10 +33,54 @@ def create
     end
   end
 
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = t("users.updated")
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t("users.deleted")
+      redirect_to users_url
```

Comment:
nên đưa xuống dưới end
```suggestion
      if @user.destroy
      ...
      else
      ...
      end
      redirect_to users_url
```